# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332787692 18000
#      Mon Mar 26 13:48:12 2012 -0500
# Node ID f537eb145f63cab8a201227ea98ad20877641d8f
# Parent  18830790deb4dd8857f93427ba7794f81709665d
cmake: set defeault gklib and metis paths to the current source directory to comply with new cmake policy

diff -r 18830790deb4 -r f537eb145f63 CMakeLists.txt
--- a/CMakeLists.txt	Tue Mar 20 11:59:44 2012 -0500
+++ b/CMakeLists.txt	Mon Mar 26 13:48:12 2012 -0500
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 2.8)
 project(ParMETIS)
 
-set(GKLIB_PATH METIS/GKlib CACHE PATH "path to GKlib")
-set(METIS_PATH METIS CACHE PATH "path to METIS")
+set(GKLIB_PATH ${CMAKE_CURRENT_SOURCE_DIR}/metis/GKlib CACHE PATH "path to GKlib")
+set(METIS_PATH ${CMAKE_CURRENT_SOURCE_DIR}/metis CACHE PATH "path to METIS")
 
 # Search for MPI.
 # GK commented this out as it seems to be creating problems
 # include(FindMPI)
 # if(NOT MPI_FOUND)
